### PR TITLE
Feature/test utils

### PR DESCRIPTION
**Pull request by ashikvarma11**
## Description
Added unit tests for `sum` and `slugify` functions.
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (security config, docs update, etc.)
- [x] Added unit tests for charts.js functions 
## Screenshot of change (if there's UI update)

### CODE REVIEW

File: lib/controller.js [hunk 1]
```diff
@@ -9,8 +9,9 @@ const { GITHUB_API_URL } = require('./config/api');
 
 // load utils
 const { emojiGenerator } = require('./public/js/emoji_generator');
-const { getColorsForLanguages } = require('./public/js/language_colors_generator');
-
+const {
```
Comment (wildan3105) [inline lib/controller.js:12]: please don't commit unecessary changes

File: lib/public/css/screen/base.min.css [hunk 2]
```diff
@@ -3,4 +3,4 @@
...
...stify-content-sm-around{justify-content:space-around !important}.align-items-sm-start{align-items:flex-start !important}.align-items-sm-end{align-items:flex-end !important}.align-items-sm-center{align-items:center !important}.align-items-sm-baseline{align-items:baseline !important}.align-items-sm-stretch{align-items:stretch !important}.align-content-sm-start{align-content:flex-start !important}.align-content-sm-end{align-content:flex-end !important}.align-content-sm-center{align-content:center !important}.align-content-sm-between{align-content:space-between !important}.align-content-sm-around{align-content:space-around !important}.align-content-sm-stretch{align-content:stretch !important}.align-self-sm-auto{align-self:auto !important}.align-self-sm-start{align-self:flex-start !important}.align-self-sm-end{align-self:flex-end !important}.align-self-sm-center{align-self:center !important}.align-self-sm-baseline{align-self:baseline !important}.align-self-sm-stretch{align-self:stretch !important}}@media (min-width: 768px){.flex-md-row{flex-direction:row !important}.flex-md-column{flex-direction:column !important}.flex-md-row-reverse{flex-direction:row-reverse !important}.flex-md-column
```
Comment (wildan3105) [inline lib/public/css/screen/base.min.css:6]: why is this file changed?

File: lib/public/js/chart.js [hunk 3]
```diff
@@ -149,7 +140,14 @@ const toggleColor = () => {
     // eslint-disable-next-line no-undef
     singleColor = chart({ limit: 20, singleColor: !singleColor });
     const btn = document.getElementById('btn-toggle');
-
     btn.classList.toggle('active');
     btn.classList.toggle('inactive');
 };
+
+(typeof module !== 'undefined' && module !== null ? module : {}).exports = {
```
Comment (wildan3105) [inline lib/public/js/chart.js:147]: can't we just do `module.exports` ?

File: package.json [hunk 4]
```diff
@@ -92,15 +92,16 @@
     "gulp": "^4.0.0",
     "gulp-rename": "^2.0.0",
     "gulp-sass": "^3.1.0",
+    "jsdom": "^16.4.0",
```
Comment (wildan3105) [inline package.json:95]: we don't need this, right? please delete

File: package.json [hunk 5]
```diff
@@ -92,15 +92,16 @@
...
     "lodash": "^4.17.19",
     "method-override": "^2.3.5",
+    "mocha": "^6.2.2",
```
Comment (wildan3105) [inline package.json:98]: please don't put this into `dependencies` cause this is only needed for unit test. can revert to the current state

**Review by wildan3105 — CHANGES_REQUESTED** [review]
Put some comments. Please don't commit unnecessary changes.

File: lib/public/js/chart.js (hunk 3 above)
Comment (ashikvarma11) [reply lib/public/js/chart.js:147]: using module.exports gave an error in the console while running the application. `module is not defined`.

**Comment by ashikvarma11** [comment]
I'll create a new PR.

**Comment by ashikvarma11** [comment]
Can I close this PR?

**Comment by wildan3105** [comment]
> Can I close this PR?

Yeah sure
